package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "build": "npm run build --workspaces",
     "build:lib": "npm run build -w libraries",
+    "test:unit": "npm run test -w libraries/ui-library",
```

Comment:
How about replacing this with `npm run test --workspaces --if-present`, which would run all tests for all libraries and examples if any these have a test script defined. This would make sure that all tests are run if we start to write tests for other packages.

Reply:
Since the other libs have the test setup not properly configured they fail the build if I add this change. I would suggest we add this once we setup the testing  configuration for the other libraries.